src/main/java/view/ListView.java

```diff
@@ -254,31 +252,36 @@ void removeSelectedMovie() {
      * Moves selected movie to a different list.
      */
     private void moveSelectedMovie() {
-        final int selectedRow = mediaListTable.getSelectedRow();
-        if (selectedRow == -1) {
-            System.err.println("No movie selected. Cannot move.");
+        if (mediaListTable == null || listViewModel == null || mediaCollectionSelector == null
+                || moveController == null) {
+            System.err.println("Initialization error. Required components are missing.");
         }
         else {
-            final ListState state = listViewModel.getState();
-            final String currentCollectionName = state.getCurrentCollectionName();
-            final Object movieNameObject = mediaListTable.getValueAt(selectedRow, 0);
-            if (movieNameObject == null) {
-                System.err.println("Movie name is null. Cannot move.");
+            final int selectedRow = mediaListTable.getSelectedRow();
+            if (selectedRow == -1) {
+                System.err.println("No movie selected. Cannot move.");
```

Comment:
maybe add a UI prompt to inform the user that their action failed

Reply:
ok Just updated a panel for all fail situations for the move button